src/plugins/kbn_vislib_vis_types/public/controls/point_series_options.html

```diff
@@ -21,8 +21,8 @@
                ng-model="vis.params.yAxis.max"
                ng-required="vis.params.setYExtents">
       </label>
-      <div ng-show="vis.params.yAxis.min > vis.params.yAxis.max">
-        <span class="text-danger">Min must not exceed max</span>
+      <div ng-show="vis.params.yAxis.max > vis.params.yAxis.min">
```

Comment:
Why did you change max and min here? This causes the error to show when the inputs are valid, and not show when it is invalid:

![screen shot 2015-09-14 at 4 10 29 pm](https://cloud.githubusercontent.com/assets/1250387/9864137/399610a6-5afb-11e5-9510-a80248fc5c34.png)

![screen shot 2015-09-14 at 4 11 29 pm](https://cloud.githubusercontent.com/assets/1250387/9864147/53f95a7a-5afb-11e5-8969-57fd956bf7ad.png)